packages/framework/src/Migrations/Version20250214133657.php

```diff
@@ -7,16 +7,14 @@
 use Doctrine\DBAL\Schema\Schema;
 use Shopsys\MigrationBundle\Component\Doctrine\Migrations\AbstractMigration;
 
-class Version20231020173331 extends AbstractMigration
+class Version20250214133657 extends AbstractMigration
 {
     /**
      * @param \Doctrine\DBAL\Schema\Schema $schema
      */
     public function up(Schema $schema): void
     {
-        $this->sql('INSERT INTO setting_values (name, domain_id, value, type) VALUES
-            (\'imageStructureMigratedForProxy\', 0, \'false\', \'boolean\')
-        ');
+        $this->sql('DELETE FROM setting_values WHERE name = \'imageStructureMigratedForProxy\'');
```

Comment:
I would also remove the entry for `Version20231020173331` from the migrations database table (otherwise, there will be a warning about a missing migration)

```
DELETE FROM "migrations"
WHERE "version" = 'Shopsys\FrameworkBundle\Migrations\Version20231020173331';
```